styleguide/source/assets/scss/03-organisms/_ipe-actions.scss

```diff
@@ -0,0 +1,151 @@
+/* Styling for the IPE Actions blocks when viewed in the AMA theme. */
+
+.contextual-region {
+  &:hover div.ipe-actions {
+    background: #f9f9f9;
+  }
+}
+
+// Specify the div selector to override default styling from panels-ipe.css.
+div.ipe-actions {
```

Comment:
i don't need we need div.ipe-actions. this can be .ipe.actions

Reply:
See the comment on line 9 which explains the specificity of the selector